libs/coin-bitcoin/src/signer.ts

```diff
@@ -0,0 +1,82 @@
+import { CryptoCurrency } from "@ledgerhq/types-cryptoassets";
+
+export type BitcoinXPub = string;
+export type BitcoinAddress = {
+  publicKey: string;
+  bitcoinAddress: string;
+  chainCode: string;
+};
+export type BitcoinSignature = {
+  v: number;
+  r: string;
+  s: string;
+};
+export type AddressFormat = "legacy" | "p2sh" | "bech32" | "bech32m" | "cashaddr";
+
```

Comment:
First,
I guess that the methods in BitcoinSigner come from ledgerjs/packages/hw-app-btc/src/Btc.ts

So, do you want the "Btc" class in "Btc.ts" inherits also this interface?

Second,
we are thinking about update the Btc.ts with a new interface in the future and the new interface is here:

https://github.com/LedgerHQ/app-bitcoin-new/blob/develop/bitcoin_client_js/src/lib/appClient.ts

because the APIs are more powerful and it supports the feature "wallet policy"

but we need to keep the old interface for altcoin because of the compatibility of nano app.
the methods of new interface are no compatible with the old one.
What do you think about this change in the future

Reply:
For the time being, I prefer to keep a duplicate definition of the Signer, as it is for the other coins.

I propose that we will see later together how we want to manage the separation between BtcOld and BtcNew.